pkg/rpc/response/errors.go

```diff
@@ -1,29 +1,35 @@
 package response
 
 import (
-	"errors"
 	"fmt"
-	"net/http"
 )
 
-type (
-	// Error object for outputting JSON-RPC 2.0
-	// errors.
-	Error struct {
-		Code     int64  `json:"code"`
-		HTTPCode int    `json:"-"`
-		Cause    error  `json:"-"`
-		Message  string `json:"message"`
-		Data     string `json:"data,omitempty"`
-	}
-)
+// Error object for outputting JSON-RPC 2.0 errors on the client side.
```

Comment:
`Error represents JSON-RPC 2.0 error type.`, for example. Client/server doesn't matter here.

Reply:
Fixed.